src/sql/azdata.proposed.d.ts

```diff
@@ -3776,6 +3776,11 @@ declare module 'azdata' {
 		errorCode: number;
 	}
 
+	export enum CmsDialog {
```

Comment:
This is unfortunate. Ideally this extension should require no or minimal changes to core code and certainly shouldn't have any references to cms directly. Can you post a screenshot of the dialog and a comment on why these core changes are necessary?

Reply:
The connection dialog behaves differently when prompted for a connection via the CMS dialog. We don't have a "generalized" connection dialog so I had to add this additional parameter.

Reply:
@kburtram @alanrenmsft thoughts? I know this is one of the apis we were trying to stabilize.